src/browser/api/system.js

```diff
@@ -610,6 +610,38 @@ exports.System = {
             }
         });
     },
+    getCookies: function(opts, callback, errorCallback) {
+        const { url, name } = opts;
+        if (url && url.length > 0 && name && name.length > 0) {
+            session.defaultSession.cookies.get({ url, name }, (error, cookies) => {
+                if (error) {
+                    log.writeToLog(1, `cookies.get error ${error}`, true);
+                    errorCallback(error);
+                } else if (cookies.length > 0) {
+                    const data =
+                        cookies.filter(cookie => cookie.httpOnly !== true).map(cookie => {
```

Comment:
:neckbeard: :neckbeard: could be simplified to `cookies.filter(cookie => !cookie.httpOnly)...`

Reply:
Updated